lib/spack/docs/tutorial_modules.rst

```diff
@@ -280,20 +527,26 @@ exceptions to the blacklist rules you can use ``whitelist``:
 
 .. code-block:: console
 
-  $ spack module refresh --module-type tcl -y
+  root@module-file-tutorial:/# spack module refresh --module-type tcl -y
+  ==> Regenerating tcl module files
+
 
-you'll see that now the module for ``gcc@6.2.0`` has reappeared:
+you'll see that now the module for ``gcc@7.2.0`` has reappeared:
```

Comment:
I had difficulty having gcc@7.2.0. Ran the command spack module refresh --module-type tcl -y but that module is still not appearing.

Reply:
Hmm, I went through the tutorial today and everything worked for me (using the Docker image).

Reply:
that might be the case. Like I said some of the steps I had to skip since I wasn't sure what was already set up for me and what wasn't. 

I just wanted to make a comment in case it was a bug or user incompetence